preview/infrastructure/harvester/provider.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+
+  backend "gcs" {
+    bucket = "3f4745df-preview-tf-state"
```

Comment:
What's `3f4745df`? ☺️

Reply:
It's a random string - bucket names are public and globally unique, so I prefer to add some randomness to each.